ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,50 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_test
+
+
+# result_type
+@handle_test(
+    fn_tree="functional.ivy.result_type",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        valid_axis=True,
+        max_axes_size=1,
+    ),
+    keepdims=st.booleans(),
+)
+def test_torch_result_type(
+    dtype_x_axis,
+    keepdims,
+    as_variable,
+    num_positional_args,
+    native_array,
+    container_flags,
+    with_out,
+    instance_method,
+    backend_fw,
+    fn_name,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=[False],
```

Comment:
No need to explicitly pass these values to the test function. They are handled by the `@handle_cmd_line_args` decorator.
Again, you can look at how other frontend functions have been implemented to get inspiration. Hope that helps 🙂

Reply:
There is no @handle_cmd_line_args
In "test_utility" so how can I use it in my file

Reply:
You don't have to use it explicitly. I believe there has been a change in hypothesis tests due to which the `@handle_cmd_line_arg` no longer exists. Instead, there are now global dictionaries that contain these arguments (https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/helpers/testing_helpers.py#L16)
that are then drawn by `hypothesis` when the test is run. 
All this is to say that you don't explicitly have to provide values for these variables as they are automatically assigned values during runtime. 
Then again, I believe looking at other examples should give you a good idea of how to write tests for the frontend.